docs/doc.py

```diff
@@ -33,12 +32,13 @@ def parse_variable_file(role_name, docs_file):
             docs_file.write("***")
             docs_file.write("\n\n")
 
-    variables_file.close
+    variables_file.close()
 
 
-docs_file = open("VARIABLES.md", "w")
+if __name__ == "__main__":
```

Comment:
Why do we need this if statement?

Reply:
A good practice moving from scripting to OOP. It doesn't impact any functionality though.